src/topology/homeomorph.lean

```diff
@@ -307,4 +310,13 @@ homeomorph_of_continuous_open (equiv.sigma_prod_distrib σ β).symm
 
 end distrib
 
+/--
+A subset of a topological space is homeomorphic to its image under a homeomorphism.
+-/
+def image {α β : Type*} [topological_space α] [topological_space β]
```

Comment:
Do you have `{α β : Type*} [topological_space α] [topological_space β]` in `variables`?